resources/assets/uad_lists.json

```
@@ -37511,6 +37517,18 @@
     "id": "com.motorola.android.fota",
     "list": "Carrier",
     "description": "AT&T Software Update, But by the carrier.",
+    "id": "com.coloros.alarmclock",
+    "list": "Oem",
+    "description": "Stock Clock app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
```

```suggestion
    "removal": "Recommended"
```
If removing this doesn't break anything, then why not `Recommended`? Or does it have dependencies?

---

I set it to advanced because there's no trackers or ads, like a stock aosp app.

---

Fair enough.